lib/m/parser.rb

```diff
@@ -67,8 +67,17 @@ def parse_options!(argv)
           testable.line = line
         end
 
+        opts.on '-r', 'Search provided directory recursively.' do |line|
+          p "Searching provided directory for tests recursively"
+          testable.recursive = true
+        end
+
         opts.parse! argv
       end
     end
+
+    def wildcard(type)
+      "#{testable.file}/" + (testable.recursive? ? "**/*#{type}*.rb" : "*#{type}*.rb")
```

Comment:
What about having this as an explicit if statement, even if that means having to repeat the `testable.file` part?